ansible/roles/ansible-tvault-contego-extension/tasks/object_store_service.yml

```diff
@@ -5,52 +5,25 @@
 
 - debug: msg="external s3:{{TVAULT_CONTEGO_EXT_S3}}" verbosity={{verbosity_level}}
 
-- name: Install tvault-object-store layer on centos/RedHat if Python2
-  package:
-    name:
-      - python-s3fuse-plugin-cent7
-    state: latest
-  when: >
-      ((ansible_distribution_major_version=="7" and ansible_distribution | lower =="centos" and PYTHON_VERSION=="python2") or
-      (ansible_distribution_major_version=="7" and ansible_distribution | lower =="redhat" and PYTHON_VERSION=="python2") or
-      (ansible_distribution_major_version=="8" and ansible_distribution | lower =="centos" and PYTHON_VERSION=="python2") or
-      (ansible_distribution_major_version=="8" and ansible_distribution | lower =="redhat" and PYTHON_VERSION=="python2"))
-
 - name: Install packages on compute node | centos-release-openstack-{{OPENSTACK_DIST}}
   package:
     name:
       - "centos-release-openstack-{{OPENSTACK_DIST | lower}}"
     state: latest
   when: >
-    (ansible_distribution_major_version=="8" and ansible_distribution | lower =="centos" and PYTHON_VERSION=="python3") or
-    (ansible_distribution_major_version=="8" and ansible_distribution | lower =="redhat" and PYTHON_VERSION=="python3")
-
-
-- name: Install tvault-object-store layer on centos/RedHat-8 if Python3
-  package:
-    name:
-      - python3-s3fuse-plugin
-    state: latest
-  when: >
-    (ansible_distribution_major_version=="8" and ansible_distribution | lower =="centos" and PYTHON_VERSION=="python3") or
-    (ansible_distribution_major_version=="8" and ansible_distribution | lower =="redhat" and PYTHON_VERSION=="python3")
+    (ansible_distribution_major_version>="8" and ansible_distribution | lower =="centos" and PYTHON_VERSION=="python3") or
+    (ansible_distribution_major_version>="8" and ansible_distribution | lower =="redhat" and PYTHON_VERSION=="python3")
 
-- name: Install tvault-object-store layer on Ubuntu/Debian if Python3
+- name: Install python3-s3fuse-plugin on CentOS8/RedHat-8
```

Comment:
we are already installing s3 during datmover installation, we don't need it here. remove this task

Reply:
removed